hosts

```diff
@@ -12,7 +11,6 @@ tester-ubuntu2004-64
 tester-awslinux
 tester-awslinux2
 tester-centos7-64
-tester-centos8-64
```

Comment:
To help with, centOS8 packages are not accessible anymore, we tried to use a different approach 
https://github.com/elastic/beats-tester/pull/219/commits/8a519340a9891e1ffc4c1f836b27f11404f9406f

but it did not work. Since centOS8 reached EOL so it has been removed for the shake of simplicity in the e2e tests